pkg/daemon/ceph/client/mgr.go

```diff
@@ -166,7 +171,13 @@ func mgrSetBalancerMode(context *clusterd.Context, clusterInfo *ClusterInfo, bal
 // ConfigureBalancerModule configures the balancer module
 func ConfigureBalancerModule(context *clusterd.Context, clusterInfo *ClusterInfo, balancerModuleMode string) error {
 	// Set min compat client to luminous before enabling the balancer mode "upmap"
-	err := setMinCompatClientLuminous(context, clusterInfo)
+	minCompatClientVersion := "luminous"
```

Comment:
How about we update this to our newest min version?
```suggestion
	minCompatClientVersion := "quincy"
```